src/routes/user/register/+page.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+	import type { IPageState } from "./types";
+
+	// Forms
+	const form_first = import("./1.svelte");
+	// let form_second = import('./2.svelte');
+	// let form_third = import('./3.svelte');
+
+	const pages = [
+		form_first
+		// form_second,
+		// form_third
+	];
+	// current page
+	let page = $state(0);
+
+	// The state of all of our pages
+	let pages_state = $state<IPageState[]>(new Array<{}>());
+
+	// Our handlers
+	function goToPage(_page: number) {
```

Comment:
Why is this not page?

Rename from `_page` -> `page`?

Reply:
Resolved on https://github.com/baseplate-admin/coreproject-v4-ui/pull/35/commits/fe9c99975460e589c5c69757b7f476f614557433